README.md

```diff
@@ -18,9 +18,9 @@ I recommend switching over to ratatui since tui-rs is currently unmaintained.
 ## Example
 
 ```rust
-use ansi_to_tui::IntoText;
-let buffer = std::fs::read("ascii/text.ascii").unwrap();
-let output = buffer.into_text();
+use ansi_to_tui::IntoText as _;
```

Comment:
this is a preference of mine but happy to revert

demonstrates that the item is an extension trait and avoids potential name clashes

Reply:
I've tended to avoid this in Ratatui mostly but it's no big deal.